[opt]_v_1_0_0_test_basen.Altis/description.ext

```diff
@@ -0,0 +1,21 @@
+onLoadIntroTime=false;
+onLoadMissionTime=false;
+disabledAI=0;
+debriefing=1;
+showGPS=1;
+Respawn = 3;
+RespawnDelay = 20;
+RespawnDialog = 0;
+enabledebugconsole = true;
+
+#include "rsc\RscBasicDefines.hpp"
+
+
+class CLib {
+	// Specify which modules (Mods) should get loaded for this mission
+    Modules[] = {"CLib"};
```

Comment:
Hier muss noch die OPT-Mod mit eingefügt werden. Sonst wird davon nix geladen

Reply:
die Mission heist doch Test ???
Das OPT nicht drin ist, weil da  jeder der Test will ,sein Modul eintragen muss, wenn er nicht alles laden will. Das OPT kann rein kommen wenn man mit allem fertig ist.

Reply:
Sie heißt ` [opt]_v_1_0_0_test_basen.Altis` und das ist nicht `TestMission.Altis`. Eine Versionierung im Missionsnamen fällt dank git auch weg, da sie redundant ist und der OPT-tag kann auch weg, da es ja nur eine lokale Dev-Mission ist.

Wenn jemand nur sein Modul laden will, aknn er/sie das ja immernoch anpassen. Aber so wie es jetzt ist, werden ich die Leute wundern, warum überhaupt nichts passiert. Wenn zu viel passiert, kann man das ja immernoch ändern.

Reply:
Eine Versionsnummer brauchen wir, da sobald daran gearbeitet wird( im Editor) die es mehre Versionen der Mission gibt. Also Fangen wir da ganz bei v1.0.0 an.

Reply:
Nein brauchen wir nicht. Dafür haben wir git
Die Versionsnummer zusätzich im Namen zu haben, zerstört zum Einen den ganzen git-workflow und zum Anderen führt es zu Verwirrung, sollte man mal vergessen, die Versionsnummer zu erhöhen.

Reply:
Wieso zerstört die etwas?
Ich finde das auch nicht so Sinnvoll das hier drüber laufen zu lassen.
Leute wie Oppa arbeiten nicht mit Github, die komme Total in schleudern wenn die Mission immer gleich heißt. Rückzuck wir das die Mission dann mal verwechselt und Änderungen gehen verloren oder es wir nur angenommen das die gemacht wurden. 

Wenn nun Oppa oder wer auch immer dann der Mission arbeitet, dann wir immer eine Kopie erstellt(zur Sicherheit),die Mission geht das um 1 nach oben und kommt hier ersetzt. 
So wird das bitte gemacht und nicht anderes !
Dann wird das was geändert

Reply:
Wenn du den Namen der Datei änderst, dann geht die History im Git dazu verloren.
Und die Mission wird (so wie ich das verstanden haben) dann einfach aus dem Git hier gezogen. Dementsprechend wird niemand irgendwelche Missionen verwechseln.

Davon abgesehen wird Oppa diese Mission hier niemals anfassen müssen, da das eine reine Test-Mission ist.

Reply:
die wird auch verändert werden in Zuge der Entwicklung, Einheiten austauschen,Basis neu gestalten, usw. Da will unbedingt vermeiden das da irgendwelche parallel Dinge geschehen. Deshalb eine Mission die von allen genutzt wird !